dart-impl/base/src/internal/domain_locality.c

```diff
@@ -829,15 +1009,16 @@ dart_ret_t dart__base__locality__domain__create_module_subdomains(
                    "module subdomain index:%d / num_domains:%d",
                    sd, module_domain->num_domains);
 
-    dart_domain_locality_t * subdomain = &module_domain->domains[sd];
+    module_domain->children[sd] = malloc(sizeof(dart_domain_locality_t));
+    dart_domain_locality_t * subdomain = module_domain->children[sd];
     dart__base__locality__domain__init(subdomain);
 
-    subdomain->level          = module_domain->level + 1;
-    subdomain->scope          = module_scopes[subdomain_gid_idx];
-    subdomain->relative_index = sd;
-    subdomain->global_index   = module_subdomain_gids[sd];
-    subdomain->parent         = module_domain;
-    subdomain->team           = module_domain->team;
+    subdomain->level            = module_domain->level + 1;
+    subdomain->scope            = module_scopes[subdomain_gid_idx];
+    subdomain->relative_index   = sd;
+    subdomain->global_index     = module_subdomain_gids[sd];
+    subdomain->parent           = module_domain;
+    subdomain->team             = module_domain->team;
```

Comment:
Those are singular white space changes in files that have lots of actual code changes.
Reviewing should not be harder because of that, `git diff -b` filters white space changes like these just fine.